### PR TITLE
DisplayInput Responsive

### PR DESCRIPTION
**Pull request by vintagepenguin**
Fixes Part of: https://github.com/mxenabled/mx-react-components/issues/373

## Product Full Width Design
screen shot 2016-07-19 at 3 17 06 pm

### Full Width Interacting w/ Input 
full_width_input

### Full Width Invalid Message 
full_width_invalid

### Full Width Saved Input
full_width_saved

### Full Width Hover Message 
hover_message_full_width

## Product Mobile Design
screen shot 2016-07-19 at 3 18 55 pm

## Actual Mobile Size In-Valid Message

mobile_invalid

## Actual Mobile Size Valid
mobile_hello

### CODE REVIEW

**Comment by iheartrachie** [comment]
When you break to put the label and input on new lines, add one for the hint/status text below the border. On mobile there is a very high change the input text will collide with the hint text.

File: src/components/DisplayInput.js [hunk 1]
```diff
@@ -25,35 +29,76 @@ const DisplayInput = React.createClass({
...
+                  type='text'
+                />
+                <div style={styles.hint}>
+                  <div style={styles.hintText}>
+                    {this.props.showHint && !this.props.status ? (<div>{this.props.hint}</div>) : null}
+                    {this.props.status ? (<div style={styles[this.props.status.type]}>{this.props.status.message}</div>) : null}
+                  </div>
+                </div>
+              </div>
+            )}
```
Comment (phantomxc) [inline src/components/DisplayInput.js]: You could make this whole block much more DRY with something like the following

```javascript
<div style={_isLargeOrMediumWindowSize() ? this.styles.wrapper : this.styles.wrapperSmall}>
```

**Comment by phantomxc** [comment]
What do you think about using the column component for the label and the input? The current implementation essentially just wraps it in a div that spans the width of the parent and settings up some padding / margin.

**Comment by vintagepenguin** [comment]
@iheartrachie here is the status text below the line for mobile: 

screen shot 2016-07-19 at 4 10 06 pm

**Comment by iheartrachie** [comment]
Talked with Dan about ☝️ offline and he's going to change the text to all be left justified and lined up. Just letting ya'll know since I'll be on vacation until next week so I won't be able to approve this PR.

**Comment by vintagepenguin** [comment]
### Mobile invalid text // getting saved
[https://gyazo.com/a39ebef92e4e0af91d2580b238fbe494](https://gyazo.com/a39ebef92e4e0af91d2580b238fbe494)

### Full width invalid text: 
[https://gyazo.com/b054e81934546a1323cc711216a47a23](https://gyazo.com/b054e81934546a1323cc711216a47a23)

### Hover state full width: 
[https://gyazo.com/4e7ec4e71540a130278a110de0f59346](https://gyazo.com/4e7ec4e71540a130278a110de0f59346)

**Comment by iheartrachie** [comment]
:shipit:

**Pull request merged**